.travis.yml

```diff
@@ -0,0 +1,76 @@
+sudo: required
+services:
+  - docker
+
+language: python
+
+cache:
+    directories:
+      - $HOME/.cache
+
+git:
+    depth: 100500
+
+env:
+    global:
+      - PRODUCT=tarantool-lrexlib
+    matrix:
+      - OS=el DIST=6
+      - OS=el DIST=7
+      - OS=fedora DIST=26
+      - OS=fedora DIST=27
+
+script:
+  # lrexlib tags have the following format: rel-major-minor-patch.
+  # packpack can not procced such format.
+  # That is why we have to pass VERSION and RELEASE.
+  - git describe --tags
+  - git clone https://github.com/packpack/packpack.git packpack;
+  - VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's/[^0-9-]*//g' |
+        cut -c 2- | sed 's#-#\.#g' | sed 's/\.[^.]*$//'`
+  - RELEASE=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's/[^0-9-]*//g' |
```

Comment:
@SudoBobo Are you intend to change it as we discussed in the scope of the lulpeg RPM spec PR?

Reply:
Changed.